src/js/utils.js

```diff
@@ -0,0 +1,13 @@
+const toArrayBySeparator = (string, separator = ',') => string.split(separator);
+
+const getRandomInteger = (maxValue) => Math.floor(Math.random() * ++maxValue);
+
+const delay = (timeout = 1000) =>
+  new Promise((resolve) => setTimeout(resolve, timeout));
+
+const getMaxValueByObjectKey = (objectArray, key) =>
+  objectArray.reduce((preWinner, current) =>
+    preWinner[key] > current[key] ? preWinner : current
+  )[key];
```

Comment:
```suggestion
const getMaxValueByObjectKey = (objectArray, key) => Math.max(...objectArray[key]);
```
이렇게 표현할 수 있겠네요!
그렇다면, 이걸 함수로 만들 필요가 있을까요?